salesagility/SuiteCRM
# Fixed #5016 -[language] Hard coded messages in opt in (ver. 7.10RC)

## Description



Fixes hard coded messages.

## Motivation and Context

Issue reference: #5016 

## How To Test This

1. Set email settings to "Confirmed opt-in"
2. Confirm opt-in an email and check that the response is correct.

## Types of changes


### Final checklist
